playgrounds/feature-service-maven-sb-3/docker/Dockerfile

```
@@ -1,9 +1,52 @@
+    -XX:AOTCache=application.aot \
+    -jar /application/application.jar
 
 EXPOSE 8080
 
-CMD ["java", "-jar", "/app/feature-service.jar"]
\ No newline at end of file
+# JVM options
+ENV JAVA_GC_LOG_OPTS="-Xlog:gc=info,safepoint:/tmp/gc.log::filecount=1,filesize=20M"
+ENV JAVA_AOT_CACHE_OPTS="-XX:+UseCompactObjectHeaders -XX:AOTCache=/application/application.aot"
+
+ENTRYPOINT exec java \
```

Hadolint (2.14.0)

[warning] Use arguments JSON notation for CMD and ENTRYPOINT arguments

(DL3025)